flow/client/client.go

```diff
@@ -21,18 +21,18 @@ func NewFlowClient(conn *grpc.ClientConn) *FlowClient {
 	}
 }
 
-func (f *FlowClient) Review(ctx context.Context, req *flow.ReviewRequest) (chan *flow.Issue, chan error, error) {
-	stream, err := f.pbClient.Review(ctx, req)
+func (f *FlowClient) Run(ctx context.Context, req *flow.Request) (chan *flow.Reply, chan error, error) {
```

Comment:
I'm wondering if it's better to return a channel of proto any? That way the data is completely separated from the reply. Any errors on the reply can be sent on the error channel.